scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -810,30 +810,30 @@ class ScalaTestAntTask extends Task {
    * Sets value from nested element <code>runpath</code>.
    */
   def addConfiguredRunpath(runpath: Path): Unit = {
-    for (element <- runpath.list)
+    for (element <- runpath.list.map(_.nn))
       this.runpath += element
   }
  
   /**
    * Sets value from nested element <code>testngsuites</code>.
    */
   def addConfiguredTestNGSuites(testNGSuitePath: Path): Unit = {
-    for (element <- testNGSuitePath.list)
+    for (element <- testNGSuitePath.list.map(_.nn))
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.